Library/Formula/bigloo.rb

```diff
@@ -45,4 +48,8 @@ def install
     manpages = %w[bgldepend bglmake bglpp bgltags bglafile bgljfile bglmco bglprof]
     manpages.each { |m| man1.install "manuals/#{m}.man" => "#{m}.1" }
   end
+
+  test do
+    assert_match "Bigloo (4", shell_output("bigloo --help")
```

Comment:
Sure, but it may take me a couple days to find time to do so. Want to wait on merging this until I can add it?